appservice/src/createAppService/SiteRuntimeStep.ts

```diff
@@ -24,15 +25,22 @@ export class SiteRuntimeStep extends AzureWizardPromptStep<IAppServiceWizardCont
                 ];
                 wizardContext.newSiteRuntime = (await ext.ui.showQuickPick(runtimeItems, { placeHolder: 'Select a runtime for your new Linux app.' })).data;
             } else {
-                const runtimeItems: IAzureQuickPickItem<ILinuxRuntimeStack>[] = this.getLinuxRuntimeStack().map((rt: ILinuxRuntimeStack) => {
+                let runtimeItems: IAzureQuickPickItem<ILinuxRuntimeStack>[] = this.getLinuxRuntimeStack().map((rt: ILinuxRuntimeStack) => {
                     return {
                         id: rt.name,
                         label: rt.displayName,
                         description: '',
                         data: rt
                     };
                 });
-
+                if (wizardContext.detectedSiteRuntime) {
+                    switch (wizardContext.detectedSiteRuntime) {
```

Comment:
You should be able to just sort by runtime (if it's defined) without using a switch at all

Reply:
What do you think about adding a little description saying (recommended) to anything that we sort? Something to indicate that we shifted the runtimes to the top?